src/xcode/ENA/ENA/Source/Scenes/Home/HomeCoordinator.swift

```diff
@@ -68,6 +68,7 @@ class HomeCoordinator: RequiresAppDependencies {
 				 .none:
 				rootViewController.dismiss(animated: false)
 				rootViewController.popToRootViewController(animated: false)
+				rootViewController.tabBarController?.selectedIndex = 0
```

Comment:
Seems strange to me, that we show the home tab in case of .checkIn and .healthCertificateFromNotification. Don't we have any routing in this case?

Reply:
both .checkIn and .healthCertificateFromNotification are checked in the showHome function in line 118 in the RootCoordinator inside the defer block so basiccly we show the home THEN excute the defer and show the rest

Reply:
Oh wow, its like "GOTO line 42"
Could you add a comment here?